Resources/public/js/views/universaldiscovery/ez-universaldiscoverymethodbaseview.js

```diff
@@ -97,6 +97,16 @@ YUI.add('ez-universaldiscoverymethodbaseview', function (Y) {
             },
 
             /**
+             * The location that the UDW tree will select on start
+             *
+             * @attribute multiple
+             * @type {String}
+             */
+            startingLocation: {
+                value: "",
```

Comment:
Instead of any empty string, I would put `false` as a default value to more reflect the case were there's no starting location.